emails/views.py

```diff
@@ -379,6 +379,10 @@ def _get_domain_address(to_address, local_portion, domain_portion):
             # receive an email due to the following exceptions
             try:
                 domain_address = DomainAddress.make_domain_address(user_profile.user, local_portion)
+                # update first_emailed_at if the alias was generated via email,
+                # indicating alias generation impromptu.
+                domain_address.first_emailed_at = datetime.now(timezone.utc)
+                domain_address.save()
```

Comment:
Can we move this "up" into `DomainAddress.make_domain_address` by passing another argument? Maybe a keyword argument? Something like:

`DomainAddress.make_domain_address(user_profile.user, local_portion, made_via_email=True)`

Reply:
Done 0373649b747316e871b50df42cc124d4eaaed1f9